shippable.yml

```diff
@@ -12,3 +15,11 @@ build:
    - shippable_retry pip install -Ur test-requirements.txt
 
    - pytest
+
+after_success:
+
+  - codecov --token=272fc7a8-bd55-4e0c-af2f-7d475d0f3c0a -r ansible/ansibullbot
```

Comment:
@gundalow it shouldn't require a token for public CIs, just `codecov` command should be enough, it uploads the report where needed.